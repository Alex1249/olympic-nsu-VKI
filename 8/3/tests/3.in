10
-988208 -542912 547275 331904 -933872 245942 844396 941220 -369380 963120
