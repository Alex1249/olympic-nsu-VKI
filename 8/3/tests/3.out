-988208 -933872 -542912 -369380 245942 331904 547275 844396 941220 963120
